src/components/Form/Form.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import users from '../../api/users';
+import { Todo } from '../interface/Todo';
+import { getUserById } from '../ultis/UserIdUtils';
+
+interface Props {
+  onSubmit: (todo: Todo) => void;
+}
+
+export const Form: React.FC<Props> = ({ onSubmit }) => {
+  const [title, setTitle] = useState('');
+  const [titleErrorMassege, setTitleErrorMassege] = useState('');
+
+  const [userId, setUserId] = useState(0);
+  const [userErrorMassege, setUserErrorMassege] = useState('');
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setTitleErrorMassege('');
+  };
+
+  const handleUserIdChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserId(+event.target.value);
+    setUserErrorMassege('');
+  };
+
+  const reset = () => {
+    setTitle('');
+    setUserId(0);
+
+    setTitleErrorMassege('');
+    setUserErrorMassege('');
+  };
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+    const trimmedTitle = title.trim();
+
+    if (!trimmedTitle) {
+      setTitleErrorMassege('Please enter a title');
+    }
+
+    if (!userId) {
+      setUserErrorMassege('Please choose a user');
+    }
+
+    if (!trimmedTitle || !userId) {
+      return;
+    }
```

Comment:
```suggestion
    if (!trimmedTitle) {
      setTitleErrorMassege('Please enter a title');
      return;
    }

    if (!userId) {
      setUserErrorMassege('Please choose a user');
      return;
    }
```

Reply:
With these changes, my npm linter gives errors and I have to leave it as I did

Reply:
still not resolved
![image](https://github.com/mate-academy/react_add-todo-form/assets/55200530/96f02feb-7f21-4e37-8760-914f44f5c422)

which error it gives precisely? add screenshot please, try to add blank line before `return`